firebase-functions/src/androidTest/java/com/google/firebase/functions/FirebaseFunctionsTest.java

```diff
@@ -38,6 +38,14 @@ public void testGetUrl() {
     functions = FirebaseFunctions.getInstance(app);
     url = functions.getURL("my-endpoint");
     assertEquals("https://us-central1-my-project.cloudfunctions.net/my-endpoint", url.toString());
+
+    functions = FirebaseFunctions.getInstance(app, "https://mydomain.com");
+    url = functions.getURL("my-endpoint");
+    assertEquals("https://mydomain.com/my-endpoint", url.toString());
+
+    functions = FirebaseFunctions.getInstance(app, "https://mydomain.com/foo");
+    url = functions.getURL("my-endpoint");
+    assertEquals("https://mydomain.com/foo/my-endpoint", url.toString());
```

Comment:
See my comment below, but add a test about using `useEmulator` with a custom domain.  I think it would fail right now.

Reply:
Done.